apps/language_server/lib/language_server/providers/completion.ex

```diff
@@ -1038,7 +1039,9 @@ defmodule ElixirLS.LanguageServer.Providers.Completion do
     locals_without_parens = Keyword.get(options, :locals_without_parens)
     signature_help_supported? = Keyword.get(options, :signature_help_supported, false)
     signature_after_complete? = Keyword.get(options, :signature_after_complete, true)
-    with_parens? = function_name_with_parens?(name, arity, locals_without_parens)
+
+    remote_calls? = String.contains?(prefix, ".")
```

Comment:
Please move this to context so the check is done once and not for every completion. Also consider using `match?({:dot, _, _}, Code.Fragment.cursor_context(prefix))`
```
iex(2)> Code.Fragment.cursor_context("a.")
{:dot, {:var, ~c"a"}, []}
iex(3)> Code.Fragment.cursor_context("a.a")
{:dot, {:var, ~c"a"}, ~c"a"}
iex(4)> Code.Fragment.cursor_context("A.a")
{:dot, {:alias, ~c"A"}, ~c"a"}
iex(5)> Code.Fragment.cursor_context("A.") 
{:dot, {:alias, ~c"A"}, []}
```

Reply:
TIL, do we need to exclude `a.` or `A.`... ?
```elixir
remote_calls? = match?({:dot, _, [_ | _]}, Code.Fragment.cursor_context(prefix))
```

Reply:
IMO it's good now